spec/mailers/bulk_url_upload_results_mailer_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'a bulk upload notification email' do
```

Comment:
No need to change this, but our consistent style is to omit the "RSpec":
```ruby
shared_examples 'a bulk upload notification email' do
```

Reply:
Noted.